js/interface-lists.js

```diff
@@ -506,7 +506,7 @@ function attahObservers() {
               return;
             }
 
-            if (!validate(field.value)) {
+            if (!field.value || !field.value.trim()) {
```

Comment:
what if `field.value` isn't a string?

Reply:
@squallstar hi, this variable can only be a string with empty value or not empty